docker/Dockerfile-workers

```diff
@@ -59,6 +59,7 @@ RUN dnf -y install \
     python3-devel \
     python3-pip \
     skopeo \
+    memcached \
```

Comment:
This should be removed. A separate slimmed down container image can be used for running memcached in production.

Reply:
@mprahl I am sorry I am not sure what you are suggesting here. Remove it completely so there is not memcached install in dev env? Or should I add memcached container to docker-compose? 
Not sure if you mean production - setting in our gitlab or if you meant this dev. env.

Reply:
@lipoja yeah, I'm thinking we don't want to install memcached in the same container as the worker. It'd be good to have a separate image for docker-compose though for testing. I like that idea, though it's not necessary.